### PR TITLE
Add file search notice

### PR DESCRIPTION
**Pull request by esethna**
I will add the GIF file once completed from design

@amyblais this should be queued up for the April 21 cloud release and v5.25 (May) on-prem release

### CODE REVIEW

File: notices.json [hunk 1]
```diff
@@ -72,5 +72,23 @@
...
+    "conditions": {
+      "audience": "all",
+      "clientType": "all",
+      "serverVersion": [">5.35"],
+      "displayDate": ">= 2021-04-21T00:00:00Z"
+    },
+    "localizedMessages": {
+      "en": {
+        "title": "Introducing file search",
+        "description": "Search in Mattermost now finds both relevant messages and files in your team's conversation history.",
```
Comment (cwarnermm) [inline notices.json:87]:
```suggestion
        "description": "Searching in Mattermost now finds both relevant messages and files in your team's conversation history.",
```

**Review by cwarnermm — APPROVED** [review]
Non-blocking suggestion provided inline.

**Comment by esethna** [comment]
@ogi-m GIF is added so this is ready for QA

**Comment by amyblais** [comment]
@ogi-m Kind reminder for this PR review.

**Comment by esethna** [comment]
@ogi-m @amyblais sorry I just noticed this never went out, this should have been released when the feature was released on cloud. Can we make sure in the future that this is added to the release process to ensure any in-product feature notices are shipped with the release?

**Comment by ogi-m** [comment]
@esethna I think we agreed we did not want to advertise the feature until the content extraction was completed on cloud

**Comment by esethna** [comment]
@ogi-m ah thanks, I am checking on the status of this ticket, looks like it was resolved earlier this month so I'm not sure if it's done yet: https://mattermost.atlassian.net/browse/MM-34801?focusedCommentId=111075

**Pull request closed without merging**